source/opt/decompose_initialized_variables_pass.cpp

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2019 Google Inc.
```

Comment:
I think it is suppose to be Google LLC now not Google Inc.

Reply:
That looks correct, I apparently have been using the wrong one.